### PR TITLE
[Doc][Improve] support chinese

### PR DESCRIPTION
**Pull request by JueLance**
### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by baicie** [comment]
run `mvn spotless:apply` to fix markdown style

**Comment by baicie** [comment]
cant find connector-v2/Config-Encryption-Decryption.md in ./docs/zh/concept/JobEnvConfig.md

**Comment by JueLance** [comment]
> cant find connector-v2/Config-Encryption-Decryption.md in ./docs/zh/concept/JobEnvConfig.md

Only the markdown in ./docs/zh/concept folder I have translated. The translation work of Config-Encryption-Decryption.md is still in progress, so I don't put it in ./docs/zh folder.

**Comment by JueLance** [comment]
This pr is relation to issue #6366

**Review by TyrantLucifer — DISMISSED** [review]
Thank you for your contribution. Unfortunately, you pull request has some empty files. Please double check and fix it.

**Comment by JueLance** [comment]
> Thank you for your contribution. Unfortunately, you pull request has some empty files. Please double check and fix it.

I add empty file because github ci will check dead link. I follow previous commit from others add some empty file to pass it. If other contributor want continue to translate, just overwrite it.

**Comment by davidzollo** [comment]
> > Thank you for your contribution. Unfortunately, you pull request has some empty files. Please double check and fix it.
> 
> I add empty file because github ci will check dead link. I follow previous commit from others add some empty file to pass it. If other contributor want continue to translate, just overwrite it.

good job, My suggestion is that you can add a message in the empty file. just like this,
"
ToDo(该文待翻译)
---------- ---------
我们正在补充和完善中文文档，如果您愿意贡献此文的翻译，请在 https://github.com/apache/seatunnel/issues/6378 下留言,  然后请提交一个翻译的 PR，一点一滴都是对 SeaTunnel 社区的贡献，感谢您的开源参与!  
"

**Comment by JueLance** [comment]
> > > Thank you for your contribution. Unfortunately, you pull request has some empty files. Please double check and fix it.
> > 
> > 
> > I add empty file because github ci will check dead link. I follow previous commit from others add some empty file to pass it. If other contributor want continue to translate, just overwrite it.
> 
> good job, My suggestion is that you can add a message in the empty file. just like this, " ToDo(该文待翻译)
> 
> 我们正在补充和完善中文文档，如果您愿意贡献此文的翻译，请在 #6378 下留言, 然后请提交一个翻译的 PR，一点一滴都是对 SeaTunnel 社区的贡献，感谢您的开源参与! "

Thank you for your suggestion. I removed all empty file that I submit, and changed corresponding link to English version link. And I have already submit PR to #6378.

**Review by EricJoy2048 — DISMISSED** [review]
LGTM, 

If Chinese document A is linked to other document B, but B has not been fully translated, we can link it to English document B first, so as to ensure the continuity of the document. After document B is translated, change the link address in document A to Chinese Document B.

**Review by ruanwenjun — COMMENTED** [review]
Still exist empty file `docs/zh/other-engine/spark.md`

**Comment by JueLance** [comment]
> Still exist empty file `docs/zh/other-engine/spark.md`

In English version, it's a empty file, too.

File: docs/zh/other-engine/spark.md
Comment (mosence) [inline docs/zh/other-engine/spark.md:1]: You can write this EMPTY FILE with 
```markdown
ToDo  [docs/en/other-engine/spark.md](https://github.com/apache/seatunnel/blob/dev/docs/en/other-engine/spark.md)
```

**Review by mosence — APPROVED** [review]

File: docs/zh/other-engine/spark.md
Comment (mosence) [reply docs/zh/other-engine/spark.md:1]: Or parent title link to ENGLISH file and DELETE this file.

**Comment by TyrantLucifer** [comment]
Overall LGTM, please waiting CI/CD

**Review by davidzollo — DISMISSED** [review]
+1
LGTM

**Review by mosence — APPROVED** [review]
LGMT +1

**Review by mosence — APPROVED** [review]

**Review by davidzollo — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**